vignettes/wwinference.Rmd

```diff
@@ -297,33 +309,62 @@ fit <- wwinference::wwinference(
     infection_feedback_pmf = infection_feedback_pmf,
     params = params
   ),
-  mcmc_options = get_mcmc_options(),
+  fit_opts = get_mcmc_options(iter_warmup = 100, iter_sampling = 100),
```

Comment:
This makes sense-- confirming though that it didn't result in diagnostic issues?

Reply:
But I think we should add some language around this saying that we used a smaller number of warmup and sampling iterations here as an example then we would recommend for production?

Reply:
Missed modifying that! Will reverse the change.